draft-ietf-httpbis-http2bis.xml

```diff
@@ -727,6 +728,9 @@ HTTP Frame {
             <t>
                 Receiving any frame other than <xref target="HEADERS" format="none">HEADERS</xref> or <xref target="PRIORITY" format="none">PRIORITY</xref> on
                 a stream in this state MUST be treated as a <xref target="ConnectionErrorHandler">connection error</xref> of type
+                <xref target="PROTOCOL_ERROR" format="none">PROTOCOL_ERROR</xref>. If this stream is server-initiated as described in
+                <xref target="StreamIdentifiers"/> then receiving a <xref target="HEADERS" format="none">HEADERS</xref> frame MUST also
+                be treated as a <xref target="ConnectionErrorHandler">connection error</xref> of type
```

Comment:
Feels like it could use at least one comma in all that.

Reply:
```suggestion
                <xref target="PROTOCOL_ERROR" format="none">PROTOCOL_ERROR</xref>. If this stream is server-initiated, as described in
                <xref target="StreamIdentifiers"/>, then receiving a <xref target="HEADERS" format="none">HEADERS</xref> frame MUST also
                be treated as a <xref target="ConnectionErrorHandler">connection error</xref> of type
```
?